MM2RandoLib/ChargingSpeed.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace MM2Randomizer
```

Comment:
This file appears twice. Is that an accident? If not, how can we reduce the duplication?

See also `RandomizerHost/Settings/ChargingSpeed.cs `

and this also comes up for the `PlayerSprite.cs` file.

Reply:
It is actually not by accident.  As part of decoupling the app/UI settings from the randomizer lib settings, and to reduce as many references and dependencies to the library as possible, I have two versions of these enums now. One is for UI settings, save and loading, and binding to view model UI, the other is only used internally to the library code.

Reply:
In that case, could we change the app configuration to store basically 2 things, a) the display string for the list in the UI, b) whatever we need to apply the sprite (I'm guessing just the file name of the IPS)? Seems like that would cut out the redundancy and make it easy for people to drop in new sprites. For instance, if they are working on adding new ones.

Reply:
I removed the duplicates. My idea was dumb LUL